oci-library/oci-hol/java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -46,44 +69,47 @@ Install Management Agent (If your host is Windows, skip to **For Windows** Secti
 
 1. Login as a user with `sudo` privileges.
 
-2. Navigate to the directory where you have downloaded the management agent software RPM file and run the following command to install the `RPM` file:
+2. Navigate to the directory where you have downloaded the management agent software RPM file and run the following commands to install the `RPM` file:
     ```
     <copy>
-    sudo rpm -ivh <rpm_file_name.rpm>
+    JDK_DIR=$(find /usr/java/ -name "jdk1.8*" -type d)
+    sudo JAVA_HOME="${JDK_DIR}" rpm -ivh <rpm_file_name.rpm>
     </copy>
     ```
 
-  For example: `sudo rpm -ivh oracle.mgmt_agent-<VERSION>.rpm`
+  For example: `sudo JAVA_HOME="${JDK_DIR}" rpm -ivh oracle.mgmt_agent-<VERSION>.rpm`
```

Comment:
i think this line is redundant. there is also no <VERSION> behind your oracle.mgmt_agent